### PR TITLE
Queue using stacks in C++

### LINKED ISSUE #5635: Queue using stacks
## 🚀 Feature
To implement Queue data structure using stacks in C++

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

YES

### PR DESCRIPTION
**Pull request by aish2002**
Fixes #5635 

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes

### Description

Implemented queue using 2 stacks in C++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#5635

### CODE REVIEW

**Comment by aish2002** [comment]
@ankitaggarwal23 , @HarshCasper  kindly review the PR

File: C-Plus-Plus/ds/Queue_using_Stacks.cpp [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        popping second stack and pushing into first stack
+        till second stack becomes empty. New value is then
+        pushed into the first stack
+    Dequeue operation proceeds by:
+        popping first stack and pushing into second stack
+        till first stack becomes empty. Thus first element
+        appears at the top of second stack. Second stack is
+        then popped.
+
+*/
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/ds/Queue_using_Stacks.cpp]:
```
/*  

QUEUE USING STACKS
A queue can be implemented using 2 stacks.

Enqueue operation proceeds by:
Popping second stack and pushing into first stack
till second stack becomes empty. New value is then
pushed into the first stack.

Dequeue operation proceeds by:
Popping first stack and pushing into second stack
till first stack becomes empty. Thus first element
appears at the top of second stack. Second stack is
then popped.

*/
```

rewrite it

File: C-Plus-Plus/ds/Queue_using_Stacks.cpp [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+#include <bits/stdc++.h>
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/ds/Queue_using_Stacks.cpp:19]: Add `using namespace std;` after `#include <bits/stdc++.h>`

File: C-Plus-Plus/ds/Queue_using_Stacks.cpp [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    Option : 2
+    Option : 1
+    Enter the value : 4
+    Option : 3
+    2 3 4
+    Option : 4
+*/
+/*  Time Complexity: O(n) for each operation
+    Space complexity : O(n)
+*/
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/ds/Queue_using_Stacks.cpp]: Rewrite it like that,

```
/*
SAMPLE I/O:

Choose an option
1. Enqueue
2. Dequeue
3. Display
4. Exit
Option : 1
Enter the value : 1
Option : 1
Enter the value : 2
Option : 1
Enter the value : 3
Option : 2
Option : 1
Enter the value : 4
Option : 3
2 3 4
Option : 4

Time Complexity: O(n) 
Space complexity : O(n)

*/
```

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by aish2002** [comment]
Necessary changes are made. @ankitaggarwal23  kindly re-review the PR

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**